helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -653,6 +653,13 @@ private RollingUpdateOp rollingUpdateTimedoutError(final RollingUpdateOpFactory
               RollingUpdateError.IMAGE_PULL_FAILED,
               metadata);
     }
+    if (!taskStatus.getContainerError().equals("")) {
```

Comment:
This is probably a matter of personal preference, but guava has a `isNullOrEmpty()` that this code base seems to like a lot.